.github/workflows/ci.yml

```diff
@@ -168,7 +168,7 @@ jobs:
     steps:
       - name: Create Dev Drive using ReFS
         run: |
-          $Volume = New-VHD -Path C:/uv_dev_drive.vhdx -SizeBytes 10GB |
+          $Volume = New-VHD -Path C:/uv_dev_drive.vhdx -SizeBytes 12GB |
```

Comment:
Windows test builds failed with the mysterious `LINK : fatal error LNK1318: Unexpected PDB error; LIMIT` without this